conduct.md

```diff
@@ -106,6 +94,23 @@ listed above,
   * Sébastien Doeraene, Scala Center ([sebastien.doeraene@epfl.ch](mailto:sebastien.doeraene@epfl.ch))
   * Kathleen Hayes, Lightbend ([kathleen.hayes@lightbend.com](mailto:kathleen.hayes@lightbend.com))
 
+### Examples of harassment
+
+The term “Harassment” includes “Unacceptable Behavior” described in the [Citizen Code of Conduct](http://citizencodeofconduct.org/).
+Examples from that resource, but not based on events in our community:
```

Comment:
This reads as if it wants to say "no, believe me, nothing that bad happened here". I do not think this is appropriate. I preferred the previous text which was completely neutral wrt. whether or not things did happen before or not:

> Behavior that will lead to exclusion includes the following:

Reply:
I think not clarifying this is more likely to lead to conspiracy theories. Explicit good. Implicits bad :-)

Reply:
I'm open to alternative formulation.

Reply:
I lean towards Seb's position here. I think referencing the Citizen Code of Conduct is a sufficient disclaimer.

Also, we shouldn't be too confident these things aren't happening in our community. I'm not aware of any doxing, violence, or sustained disruption of talks/events, true; but the rest of the list definitely happens in any sufficiently large tech community, and that includes ours.

Reply:
updated to that